docs/dg/dev/architecture/module-api/semantic-versioning-unstable-version.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Semantic versioning - unstable version
+description: Spryker releases early as unstable. Learn more about them in this document.
+last_updated: Apr 9, 2025
+template: concept-topic-template
+redirect_from:
+  - /docs/scos/dev/architecture/module-api/module-api.html
+related:
+  - title: Semantic versioning - major vs. minor vs. patch release
+    link: docs/scos/dev/architecture/module-api/semantic-versioning-major-vs.-minor-vs.-patch-release.html
+---
+
+# Understanding Unstable Modules in Spryker
```

Comment:
Why do you invest the new term "Unstable modules"?
You can use 0.x modules (as zero-major version).

It's described here
https://semver.org/#spec-item-4

Another type are pre-release modules
https://semver.org/#spec-item-9

Reply:
@PhilinTv I did not. This terminology was invented by someone when I've joined Spryker.